components/pedigree/resources/src/main/resources/pedigree/view/nodeMenu.js

```diff
@@ -1354,17 +1354,21 @@ define([
 
                 var suggestContainer = container.down('div.patient-newlink-container');
                 var suggestInput     = container.down('input[type=text].suggest-patients');
+                suggestInput.placeholder = " type patient name or identifier";
```

Comment:
@veronikaslc If a space looks better for the placeholder, then it probably also looks better when there's a value entered, in which case the effect should be obtained via styling (padding left & right) instead.

I don't have a strong opinion regarding small vs capital first letter, as link as it's consistent across placeholders on the same form.